include/oneapi/tbb/detail/_flow_graph_body_impl.h

```diff
@@ -279,19 +279,23 @@ class apply_body_task_bypass
     NodeType &my_node;
     Input my_input;
 
-    graph_task* call_apply_body_bypass_impl(std::true_type) {
+    using check_metainfo = std::is_same<BaseTaskType, graph_task>;
+    using not_contains_metainfo = std::true_type;
+    using contains_metainfo = std::false_type;
+
+    graph_task* call_apply_body_bypass_impl(not_contains_metainfo) {
```

Comment:
Avoiding negation?
```suggestion
    graph_task* call_apply_body_bypass_impl(missing_metainfo) {
```

Or, perhaps, even better (at least from my point of view). Though, requires changing accompanying helper:
```suggestion
    graph_task* call_apply_body_bypass_impl(without_metainfo) {
    //...
#if __TBB_PREVIEW_FLOW_GRAPH_TRY_PUT_AND_WAIT
    graph_task* call_apply_body_bypass_impl(with_metainfo) {
    //...
```
It literally reads "_call apply body impl with/without metainfo_".

Reply:
Done